include/atalk/globals.h

```diff
@@ -10,10 +10,7 @@
 #include <grp.h>
 #include <sys/types.h>
 #include <stdbool.h>
-
-#ifdef HAVE_NETDB_H
-#include <netdb.h>  /* this isn't header-protected under ultrix */
-#endif /* HAVE_NETDB_H */
```

Comment:
Last I checked there were tons more `HAVE_NETDB_H` checks elsewhere in the codebase. I wonder if all of them can be removed, or if they still serve a purpose for another OS?

Reply:
netdb.h is a widely used header in posix oses, and contains definitions for network database operations. macOS and Ubuntu have it for example. So good to keep I think...

Reply:
Right, that's what I thought!

So what I wanted to suggest is that we keep the ifdef and just remove the code comment.
I suppose the macro is there for non-posix systems that may not have this header...

```
#ifdef HAVE_NETDB_H
...
#endif /* HAVE_NETDB_H */
```

Reply:
Done.